### PR TITLE
Bump deps take 2

### PR DESCRIPTION
**Pull request by lishaduck**
I'm rewriting my whole stack.
I started by rebumping dependencies.
No production bounds are changed, as that would be breaking.
Development dependencies are updated.
Code is changed accordingly.
All commits are atomic, so that git bisect still works.

### CODE REVIEW

File: lib/app-wrapper.js [hunk 1]
```diff
@@ -2,34 +2,22 @@ const path = require('path');
 const ElmCommunication = require('./elm-communication');
 const loadCompiledElmApp = require('./load-compiled-app');
 
-try {
-  // Conditional imports, because `worker_threads` is not supported by default
-  // on Node v10
```
Comment (jfmengels) [inline lib/app-wrapper.js:7]: I was thinking of doing this soon. Nice :+1:

**Review by jfmengels — COMMENTED** [review]
Thank you very much for the changes!

File: package.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "name": "elm-review",
   "version": "2.12.0",
   "description": "Run elm-review from Node.js",
+  "type": "commonjs",
```
[the remark this answers is not in the document]
Comment (lishaduck) [reply package.json:5]: I was just making it more explicit for myself, and I think it affects how typescript does things.
Either way, node is working on switching the default, so it's better to set it.

File: lib/watch.js [hunk 3]
```diff
@@ -326,13 +324,12 @@ function createFileWatcher(
...
       let elmFile = AppState.getFileFromMemoryCache(relativePath);
-      if (!elmFile) {
-        elmFile = {
-          path: relativePath,
-          source: '',
-          ast: null
-        };
-      }
+
+      elmFile ??= {
```
[the remark this answers is not in the document]
Comment (lishaduck) [reply lib/watch.js:328]: Yeah, you could! Then, you could make it const.

**Pull request merged**